CHANGELOG.md

```diff
@@ -276,6 +272,18 @@ Non-backwards compatible changes
   exported by `Data.Rational.Base`. You will have to open `Data.Integer(.Base)`
   directly to use them.
 
+* The types of the proofs `pos⇒1/pos`/`1/pos⇒pos` and `neg⇒1/neg`/`1/neg⇒neg` in
+  `Data.Rational(.Unnormalised).Properties` have been switched, as the previous
+  naming scheme didn't correctly generalise to e.g. `pos+pos⇒pos`. For example
+  the proof `1/pos⇒pos` now has the type:
+  ```
+  ∀ p .{{_ : Positive p}} → Positive (1/ p)
+  ```
+  rather than
+  ```
+  ∀ p .{{_ : NonZero p}} .{{Positive (1/ p)}} → Positive p
```

Comment:
These are two distinct statements

Reply:
It was deliberate but very unclear! I've clarified what I meant, thanks :smile: